spanner/CHANGELOG.md

```diff
@@ -4,6 +4,21 @@
 
 [1]: https://pypi.org/project/google-cloud-spanner/#history
 
+## 1.4.0
+
+### Implementation Changes
```

Comment:
Shouldn't this be `### Bug fixes`?

Reply:
Or `### Bugs fixed`?

Reply:
Nope, our template is "Implementation Changes".